Bottom: 3f08f3889671961b64da2720a5e842c24b0f657a
Top:    5dd55bd528eddef0317d511bfda14dcf268cc6e9
Author: Ben Greear <greearb@candelatech.com>
Date:   2016-06-04 15:58:19 -0700

Refresh of wireless-print-error-logs-for

---

diff --git a/net/wireless/nl80211.c b/net/wireless/nl80211.c
index 424ce82..e3a7427 100644
--- a/net/wireless/nl80211.c
+++ b/net/wireless/nl80211.c
@@ -6104,7 +6104,7 @@ static int nl80211_trigger_scan(struct sk_buff *skb, struct genl_info *info)
 
 			if (band < 0 || band >= NUM_NL80211_BANDS) {
 				pr_err("scan:  band %d out of range, num-bands: %d\n",
-				       band, IEEE80211_NUM_BANDS);
+				       band, NUM_NL80211_BANDS);
 				err = -EINVAL;
 				goto out_free;
 			}
